cmd/juno/juno.go

```diff
@@ -231,6 +270,12 @@ func NewCmd(config *node.Config, run func(*cobra.Command, []string) error) *cobr
 	junoCmd.Flags().Uint16(wsPortF, defaultWSPort, wsPortUsage)
 	junoCmd.Flags().String(dbPathF, defaultDBPath, dbPathUsage)
 	junoCmd.Flags().Var(&defaultNetwork, networkF, networkUsage)
+	junoCmd.Flags().String(cnFeederURLF, defaultCNFeederURL, networkCustomFeederUsage)
+	junoCmd.Flags().String(cnGatewayURLF, defaultCNGatewayURL, networkCustomGatewayUsage)
+	junoCmd.Flags().Int(cnL1ChainIDF, defaultCNL1ChainID, networkCustomL1ChainIDUsage)
+	junoCmd.Flags().String(cnL2ChainIDF, defaultCNL2ChainID, networkCustomL2ChainIDUsage)
+	junoCmd.Flags().Int(cnProtocolIDF, defaultCNProtocolID, networkCustomProtocolIDUsage)
```

Comment:
looks like unused?

Reply:
Yup, cnProtocolIDF isn't needed. I've removed it.